app/migrations/versions/3500c08573ff_.py

```diff
@@ -0,0 +1,32 @@
+"""Enforcing non nullable Organisation columns
+
+Revision ID: 3500c08573ff
+Revises: 6d8a938930ff
+Create Date: 2020-04-24 15:41:41.934453
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '3500c08573ff'
+down_revision = '6d8a938930ff'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("UPDATE organisation SET _country_code = 'AU' WHERE _country_code is NULL")
```

Comment:
This will update all existing organisations with null `_country_code`, defaulting to AU